src/mainboard/protectli/vault_adl_n/variants/vp2430/gpio.c

```diff
@@ -0,0 +1,1415 @@
+
+	/* GPP_SYS_RESETB - n/a */
+	/* DW0: 0x40000702, DW1: 0x0003c000 */
+	/* DW0: PAD_BUF(TX_RX_DISABLE) | (1 << 1) - IGNORED */
+	/* PAD_CFG_NF(GPP_SYS_RESETB, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_SYS_RESETB, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_BUF(TX_RX_DISABLE) | (1 << 1), PAD_IOSSTATE(IGNORE)),
+
+	/* GPP_MLK_RSTB - n/a */
+	/* DW0: 0x40000700, DW1: 0x00000000 */
+	/* DW0: PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_MLK_RSTB, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_MLK_RSTB, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* ------- GPIO Group GPP_E ------- */
+
+	/* GPP_E0 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E0, NONE),
+
+	/* GPP_E1 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E1, NONE),
+
+	/* GPP_E2 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E2, NONE),
+
+	/* GPP_E3 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E3, NONE),
+
+	/* GPP_E4 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E4, NONE),
+
+	/* GPP_E5 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E5, NONE),
+
+	/* GPP_E6 - GPIO */
+	/* DW0: 0x44000200, DW1: 0x00000000 */
+	PAD_CFG_GPO(GPP_E6, 0, DEEP),
+
+	/* GPP_E7 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E7, NONE),
+
+	/* GPP_E8 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E8, NONE),
+
+	/* GPP_E9 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E9, NONE),
+
+	/* GPP_E10 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E10, NONE),
+
+	/* GPP_E11 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E11, NONE),
+
+	/* GPP_E12 - GPIO */
+	/* DW0: 0x44000300, DW1: 0x00000000 */
+	PAD_NC(GPP_E12, NONE),
+
+	/* GPP_E13 - GPIO */
+	/* DW0: 0x40880102, DW1: 0x00000000 */
+	/* DW0: (1 << 1) - IGNORED */
+	/* PAD_CFG_GPI_SCI(GPP_E13, NONE, DEEP, LEVEL, INVERT), */
+	_PAD_CFG_STRUCT(GPP_E13, PAD_RESET(DEEP) | PAD_IRQ_ROUTE(SCI) | PAD_RX_POL(INVERT) | PAD_BUF(TX_DISABLE) | (1 << 1), 0),
+
+	/* GPP_E14 - DDSP_HPDA */
+	/* DW0: 0x44000700, DW1: 0x00024000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_E14, NONE, DEEP, NF1), */
+	_PAD_CFG_STRUCT(GPP_E14, PAD_FUNC(NF1) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), PAD_IOSSTATE(TxDRxE)),
+
+	/* GPP_E15 - Reserved */
+	/* DW0: 0x44000b00, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_E15, NONE, DEEP, NF2), */
+	_PAD_CFG_STRUCT(GPP_E15, PAD_FUNC(NF2) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
+
+	/* GPP_E16 - Reserved */
+	/* DW0: 0x44000b00, DW1: 0x00000000 */
+	/* DW0: PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE) - IGNORED */
+	/* PAD_CFG_NF(GPP_E16, NONE, DEEP, NF2), */
+	_PAD_CFG_STRUCT(GPP_E16, PAD_FUNC(NF2) | PAD_RESET(DEEP) | PAD_TRIG(OFF) | PAD_BUF(TX_RX_DISABLE), 0),
```

Comment:
These are NC per shcematics

Reply:
[NC'd](https://github.com/Dasharo/coreboot/pull/576/commits/68fe32358db1c4826321169730cb09ec89ec4a40#diff-2f9e6fbf9293a4dbeecc2314eb0b449c4eb64691bfc1c58981fc506a602441bcR746)

Reply:
You changed to PAD_CFG_NF not PAD_NC

Reply:
oof sorry, [fixed](https://github.com/Dasharo/coreboot/pull/576/commits/44bdda95a261333ce63234b7cf456a35843ae23a)